codecov.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# For documentation: https://docs.codecov.io/docs/codecovyml-reference
+# Incremental settings: https://codecov.io/gh/twisted/incremental/settings/yaml
+#
+# We want 100% coverage for new patches to make sure we are always increasing
+# the coverage.
+#
+codecov:
+  require_ci_to_pass: yes
+  notify:
+    # We have at least 10 builds in GitHub Actions and 12 in Azure
+    # and lint + mypy + docs + ReadTheDocs
+    after_n_builds: 15
+    wait_for_ci: yes
+
+coverage:
+  precision: 2
+  round: down
+  status:
+    patch:
+      default:
+        # New code should have 100% CI coverage as the minimum
+        # quality assurance measurement.
+        # If there is a good reason for new code not to have coverage,
+        # add inline pragma comments.
+        target: 100%
+    project:
+      default:
+        # Temporary allow for a bit of slack in overall code coverage due to
+        # swinging coverage that is not triggered by changes in a PR.
+        # See: https://twistedmatrix.com/trac/ticket/10170
```

Comment:
Does this apply to incremental?

Reply:
I think that this is valid for any project... due to

>  There is also codecov, that will report a decrease in project coverage when a line is removed